bundles/org.openhab.binding.goecharger/src/main/java/org/openhab/binding/goecharger/internal/handler/GoEChargerHandler.java

```diff
@@ -298,10 +316,10 @@ private String getWriteUrl(String key, String value) {
 
     private void sendData(String key, String value) {
         String urlStr = getWriteUrl(key, value);
-        logger.trace("POST URL = {}", urlStr);
+        logger.trace("GET URL = {}", urlStr);
 
         try {
-            HttpMethod httpMethod = HttpMethod.POST;
+            HttpMethod httpMethod = HttpMethod.GET;
```

Comment:
Just curious: Does this mean that it didn't work previously, or does it mean that both GET and POST will work? In other words: What is the reason for this change?

Reply:
I took this change from the other PR, I will try to test it with my charger with hardware revision 3 and API v1

Reply:
working both with GET and POST